salt/metalk8s/orchestrate/deploy_new_node.sls

```diff
@@ -0,0 +1,65 @@
+{%- if pillar['bootstrap_id'] %}
+{%-   set control_plane_ips = salt.saltutil.runner('mine.get', tgt=pillar['bootstrap_id'], fun='control_plane_ip') %}
+{%- else %}
+{%-   set control_plane_ips = {} %}
+{%- endif %}
+
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%-   set control_plane_ip = control_plane_ips[pillar['bootstrap_id']] %}
+{%- else %}
+{%-   set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+
+Deploy salt-minion on a new node:
```

Comment:
We should only do this when the minion isn't known to the master yet I think, and if it already 'exists', apply the `metalk8s.roles.salt-minion` on it.

However.

The role was `minion`, not `salt-minion`, and I'd like to keep it that way TBH.

Reply:
For roles it's done.
I will try to do something to only execute this if you don't have salt

Reply:
I will create another ticket about `deploy_new_node` orchestrate lets merge this PR when all other comments get fixed

Reply:
#1017